src/vue/modules/assets/shared/components/balance-viewer.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    class="balance-viewer__balances-wrp--direction-column"
```

Comment:
1. invalid use of the validator
2. it is recommended to make the root element be BEM’s block. Exceptions are possible but it is not the case.

Also, it is used and can be used by Passport only so `passport-balances` would be fine. The block is not modified in any way so pls drop that modifier. Update the file name accordingly